playbooks/container/build-publish-tag.yml

```diff
@@ -0,0 +1,76 @@
+---
+- name: Build and Push Container Image
+  hosts: "all"
+
+  environment:
+    registry: "{{ docker_registry | default('registry.scs.community') }}"
+    project: "{{ registry_project }}"
+    repository: "{{ image_repository }}"
+    container_name: "{{ container_name | default( zuul.project.short_name ) }}"
+    containerfile_path: "{{ containerfile_path | default('.') }}"
+    version: "{{ zuul['tag'] | default('latest') }}"
+
+  tasks:
+    - name: "Log into registry"
+      community.docker.docker_login:
+        registry_url: "{{ docker_registry }}"
+        username: "{{ secret.DOCKER_USERNAME }}"
+        password: "{{ secret.DOCKER_PASSWORD }}"
+      when: push_image | bool
+
+    - name: "Run build script"
+      ansible.builtin.shell:
+        executable: "/bin/bash"
+        chdir: "{{ zuul.project.src_dir }}"
+        cmd: |
+          set -e
+          set -o pipefail
+          set -x
+
+          created=$(date --rfc-3339=ns)
+          revision=$(git rev-parse --short HEAD)
+
+          docker buildx build \
+              --label "org.opencontainers.image.created=${created}" \
+              --label "org.opencontainers.image.documentation=https://docs.scs.community" \
```

Comment:
`org.opencontainers.image.documentation` should maybe be configurable, to point to the specific projects documentation.

Reply:
Made configurable